lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -27,6 +27,9 @@
 
 /** @typedef {import('./dom')} DOM */
 
+const VIEWER_ORIGIN = 'http://localhost:8000';
```

Comment:
Need a good way to control when to use localhost vs when to use the hosted app. Important for local development. Any ideas?